docs/clients/javascript/core.md

```diff
@@ -0,0 +1,107 @@
+
+## **Considerations**
+
+**Maximal security and decentralization**: The Ceramic Core client does not have trusted relationships with any external nodes. With Ceramic Core, streams that are [written](../../build/writes.md), [queried](../../build/queries.md), or [pinned](../../build/pinning.md) are verified in the local environment which is great if you need maximal security and decentralization in your application. 
+
+**Transitory data availability**: Streams created with Ceramic Core will only be available on the network as long as this node remains online. For example for setups that use the Core Client directly in-browser, when your user closes the tab any stream created by that user will become unavailable  on the network until the user opens the application again. For more resilient data availability you can always replicate and pin streams on secondary long-running nodes, or instead use the [JS HTTP Client](./http.md) which relies on a remote node more likely to always be online.
+
+**Setup complexity**: You will need to configure an [IPFS](../../learn/glossary.md#ipfs) node which supports the [dag-jose](../../learn/glossary.md#dagjose) data format and ensure connectivity to the rest of the Ceramic network. See [installation](#installation) below for instructions on how to do this.
+
+**Swap for JS HTTP Client at any time**: The JS Core Client and the [JS HTTP Client](./http.md) implement the same [CeramicApi](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.ceramicapi-1.html){:target="_blank"} TypeScript interface, so swapping between clients is seamless and doesn't require changing your application logic; it only requires changing your setup.
+
+## **Installation**
+Installing the JS Core Client requires a console, [Node.js](https://nodejs.org/en/){:target="_blank"} v14, and [npm](https://www.npmjs.com/get-npm){:target="_blank"} v6. Make sure to have these installed on your machine.
+
+    !!! warning ""
+    While npm v7 is not officially supported, you may be able to get it to work anyway, however you will need to have the `node-pre-gyp` package installed globally:
+    
+    ```bash
+    npm install -g node-pre-gyp
+    ```
+    
+    *This is required until node-webrtc (which IPFS depends on) [is upgraded](https://github.com/node-webrtc/node-webrtc/pull/694).*
+
+### 1. Install the Core client
+Open your console and install the JS Core Client using npm.
+
+``` bash
+npm install @ceramicnetwork/core
+```
+
+### 2. Import the Core client
+
+``` javascript
+import Ceramic from '@ceramicnetwork/core'
+```
+
+### 3. Import IPFS with dag-jose
+Ceramic utilizes the [dag-jose](../../learn/glossary.md#dagjose) IPLD codec to format and store data in IPFS.
+
+``` javascript
+import IPFS from 'ipfs'
+import dagJose from 'dag-jose'
+import { sha256 } from 'multiformats/hashes/sha2'
```

Comment:
This should be updated to the new way of doing it. cc @stbrody

Reply:
Fixed